### PR TITLE
update acknowledgements/sponsorship

### PR DESCRIPTION
**Pull request by orbeckst**
- CBP added to acknowledgements
- CBP logo in logos/ and in README
- made clear that MolSSI is primary sponsor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,20 +121,27 @@ Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
 
 Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop materials.
 
-We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
+We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI), who is the primary sponsor of this workshop.
```
Comment (orbeckst) [inline README.md]: @armcdona would you like to change anything here?

File: README.md [hunk 2]
```diff
@@ -121,20 +121,27 @@ Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
...
 
 <img
 src="/logos/MolSSI_Logo.png"
 title="Molecular Sciences Software Institute Logo" alt="Molecular Sciences Software Institute Logo"
 style="float: left; height: 5em; " />
 
-This workshop has been made possible in part by a [grant](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) from the Chan Zuckerberg Initiative (CZI) DAF, a donor advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989). MDAnalysis also thanks NumFOCUS for its continued support as our fiscal sponsor.
+This workshop has been made possible in part by a [grant](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) from the Chan Zuckerberg Initiative (CZI) DAF, a donor advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989) and through the support of the [Center for Biological Physics](https://cbp.asu.edu/) at Arizona State University.
```
Comment (orbeckst) [inline README.md:131]: @jennaswa @fiona-naughton @yuxuanzhuang @IAlibay are we ok with mentioning CBP here and add the logo below?

**Review by yuxuanzhuang — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jennaswa) [inline README.md]:
```suggestion
We would like to give a special thanks to our partner and primary sponsor for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
```

Comment (jennaswa) [reply README.md]: A suggestion for reducing the number of words that you are welcome to take or leave

**Review by jennaswa — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -121,20 +121,27 @@ Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
 
 Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop materials.
 
-We would like to give a special thanks to our partner for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
+We would like to give a special thanks to our partner and primary sponsor for this workshop, the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI).
```
Comment (orbeckst) [inline README.md:124]: @armcdona would you like to change anything here?

Comment (armcdona) [reply README.md:124]: This looks good to me.

Comment (orbeckst) [reply README.md:124]: thanks!

**Pull request merged**